### ISSUE TITLE
Instant Reload/Auto Reload

### ISSUE THREAD

**Issue by evanrich2404** [ritwickdey/vscode-live-server#2923]
## I'm submitting a...

```html
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ x ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Feature request
[ ] Documentation issue or request
[ ] Other: <!-- Please describe: -->
```

## Current behavior

I am working on my webpage, getting buttons setup to dynamically generate card modals with info in them. No matter what I changed, it would instantly refresh as soon as I clicked any of the buttons. I have found a different way to preview the webpage and found that we weren't doing anything wrong with our code, and it displays as expected outside your extension. I tried many different things but was going crazy and about to start pulling hairs and maybe teeth because my code looked like it should work, but it would refresh right after I saw the data and cards start populating. 

TLDR:
It keeps refreshing right after clicking a button to display more information...

## Expected behavior

It really should just let me use it like a normal website instead of refreshing as soon as you try to do anything. We don't even have a tonne of functionality yet. Our first big feature looked broken, but it was just this extension.

## Environment

Windows 11, WSL2, Docker, (docker shouldn't be touching the frontend but included it)

```html
Browser:

- [ ] Chrome (desktop) version XX
- [ x ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] IE version XX
- [ ] Edge version XX

For Tooling issues:

- Live Server: <!-- Check which version is installed --> 
- 5.7.9
- Platform:  <!-- Mac, Linux, Windows -->
- Windows 11 (WSL2 etc..)
- Visual Studio Code: <!-- `code --version` -->
- 1.88.1
```

## Others

Really have been liking your extension, but this issue drove me mad for over a day!
Thank you for taking your time to look at my report!
